knife-ec2.gemspec

```diff
@@ -16,7 +16,7 @@ Gem::Specification.new do |s|
   s.required_ruby_version = ">= 2.5"
 
   s.add_dependency "chef", ">= 15.1"
-  s.add_dependency "fog-aws", ">= 1", "< 4"
+  s.add_dependency "aws-sdk", "~> 3.0"
```

Comment:
So one thing we're going to want to do is use the individual aws-sdk gems instead of the main gem. The main gem  depends on 182 other gems which we really don't want to do in Chef-DK. A good start would be to see if we can get away with just the ec2 gem.

Reply:
@tas50 Yes, we can do that. As per my understaning, we just only need 'aws-sdk-ec2' and `aws-sdk-s3`. will update and let you know in case of any issue thanks.